src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  people: Person[];
+  selectedPersonSlug: string;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  selectedPersonSlug,
+}) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            name,
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+            slug,
+          } = person;
+
+          const isMother = people.filter(personSearch => {
+            return personSearch.name === motherName;
+          })[0];
+
+          const isFather = people.filter(personSearch => {
+            return personSearch.name === fatherName;
+          })[0];
```

Comment:
Why do you need to use a filter here? Seems that the find method will be more appropriate. By the way, what will be if there is no mother or father? And `is` at the beginning means that it'll be a boolean type. But it's not, so rename it
```
const mother = people.find(personSearch => personSearch.name === motherName) || null;